app/controllers/event/participation_lists_controller.rb

```diff
@@ -13,12 +13,17 @@ class Event::ParticipationListsController < SimpleCrudController
 
   def create
     new_participations = build_new_participations
-    ActiveRecord::Base.transaction do
-      new_participations.map(&:save).all?(&:present?)
+    success = ActiveRecord::Base.transaction do
+      new_participations.map(&:save!).all?
```

Comment:
Die vorherige Implementation führte meiner Meinung nach nicht zu einem Rollback, wenn etwas fehlgeschlagen hat. Allenfalls war das aber auch gewollt, aber dann müsste es nicht in eine Transaction gewrappt werden, oder?